resources/knative-build/README.md

```diff
@@ -0,0 +1,9 @@
+# Knative Build
+
+## Overview
+
+This module is still experimental.
+
+Build is custom resource provided by knative. This can be used for building container images from the source code and push to docker registry of preference. This pushed docker image can be later pulled by the kantive-serving to start lambdas. For more details please refer to [knative build](https://github.com/knative/docs/tree/master/docs/build)
```

Comment:
```suggestion
Build is a custom resource provided by knative. This can be used for building container images from the source code and push to a docker registry of preference. This pushed docker image can be later pulled by knative-serving to start lambdas. For more details please refer to [knative build](https://github.com/knative/docs/tree/master/docs/build)
```

Reply:
fixed.